### ISSUE TITLE
sha1 pcr?

### ISSUE THREAD

**Issue by hoxitfoxave** [kkamagui/bitleaker#17]
I am attempting to recover my personal data. I first used napper, which reports my system is vulnerable. However when I run bitleaker, I get the following "Get PCR data from bitleaker driver.. Fail". Does the bitleaker driver support sha1 pcr data, and is it possible to recover my data using bitleaker?

![img](https://github.com/kkamagui/bitleaker/assets/133965521/9796d6b7-8493-4910-88be-567014fdcc83)

**Comment by oliveskur** [comment]
Did you run ./bootstrap before using Bitleaker?

**Comment by hoxitfoxave** [comment]
> Did you run ./bootstrap before using Bitleaker?
Yes, I have. I tried it on a different pc that uses sha256 and it successfully executes "Get PCR data from bitleaker driver."
However, my bitLocker-locked pc that I am attempting to recover data from uses sha1 and fails with "Get PCR data from bitleaker driver.. Fail"

**Comment by hoxitfoxave** [comment]
> Did you run ./bootstrap before using Bitleaker?

Yes, I have. I tried it on a different pc that uses sha256 and it successfully executes "Get PCR data from bitleaker driver."
However, my bitLocker-locked pc that I am attempting to recover data from uses sha1 and fails with "Get PCR data from bitleaker driver.. Fail"
